x-pack/plugins/synthetics/public/legacy_uptime/components/monitor_management/add_monitor_btn.tsx

```diff
@@ -120,7 +133,9 @@ export const AddMonitorBtn = () => {
           <EuiButton
```

Comment:
We need to enrich this tooltip content for more scenarios in which this could be disabled.